package.json

```
@@ -15,60 +15,60 @@
   "engines": {
     "node": "16.15.1",
     "npm": "Use the Yarn Classic (1.x) CLI instead of npm",
-    "yarn": "1.22.18"
+    "yarn": "1.22.19"
   },
   "volta": {
     "node": "16.15.1",
-    "yarn": "1.22.18"
+    "yarn": "1.22.19"
   },
   "dependencies": {
-    "@angular/animations": "14.0.0",
-    "@angular/common": "14.0.0",
-    "@angular/compiler": "14.0.0",
-    "@angular/core": "14.0.0",
-    "@angular/forms": "14.0.0",
-    "@angular/platform-browser": "14.0.0",
-    "@angular/platform-browser-dynamic": "14.0.0",
-    "@angular/router": "14.0.0",
-    "@nrwl/angular": "14.2.0-rc.2",
-    "rxjs": "7.4.0",
+    "@angular/animations": "14.0.2",
+    "@angular/common": "14.0.2",
+    "@angular/compiler": "14.0.2",
+    "@angular/core": "14.0.2",
+    "@angular/forms": "14.0.2",
+    "@angular/platform-browser": "14.0.2",
+    "@angular/platform-browser-dynamic": "14.0.2",
+    "@angular/router": "14.0.2",
+    "@nrwl/angular": "14.3.6",
+    "rxjs": "7.5.5",
```

issue: Let's intentionally pin RxJS to 7.4.x since that is the earliest RxJS 7.x version officially supported by Angular 14.
```suggestion
    "rxjs": "7.4.0",
```